lib/ex_machina.ex

```diff
@@ -58,6 +58,17 @@ defmodule ExMachina do
       def create_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.create_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def factory(factory_name, attrs \\ [])
```

Comment:
Oh I see. I think we should just remove the ability to define a function without the `attrs` param. So we can delete the test on line 41 of `ex_machina_test.exs`. Since we have the macro there will always be a factory/2 function